components/project-card/meta/get-involved.jsx

```diff
@@ -62,8 +72,9 @@ class GetInvolved extends React.Component {
       return null;
 
     return (
-      <div className="get-involved pb-3 mb-3">
-        <h2>Help needed</h2>
+      
```

Comment:
Extra line here.

---

Note that prettier/eslint is set up on this project but I made it to only raise warnings instead of errors otherwise it will make code review hard (hard to distinguish code changes vs linting fixes). I'm planning to fix all linting errors in the codebase once major updates to Pulse are done. In the meantime... it'd be awesome if we can make sure the new code we are adding to the codebase is lint error free! 😉 

I found using the "fix this prettier problem" option on VSCode helpful (for now let's avoid using "fix all").

<img width="319" alt="image" src="https://user-images.githubusercontent.com/2896608/63203893-6ae34100-c047-11e9-9de6-f4933ae4099f.png">

Reply:
hmmm, I don't have that option. How'd you get those options?
<img width="945" alt="Screen Shot 2019-08-19 at 12 56 07 PM" src="https://user-images.githubusercontent.com/36484608/63284068-c8ae9d80-c280-11e9-9933-eb8ec60a3b47.png">

Reply:
It could be from some plug-in I installed.

![prettier](https://user-images.githubusercontent.com/2896608/63298809-33fc6d80-c289-11e9-9cdc-c3eb22a8c05d.gif)

If this is troublesome we can fix linting errors after all major updates to Pulse are done. For now just fix the indentation issues (re: spaces vs tabs) that I mentioned in my earlier comments.

Reply:
yeah, i fixed them. I used to see that little light bulb, but not anymore ugh 🙄